network/proto/network.proto

```diff
@@ -49,16 +63,16 @@ message Close {
         Node node = 1;
 }
 
-// Solicit is sent when requesting route advertisement from the network nodes
+// Solicit is sent when soliciting routes from the network nodes
 message Solicit {
```

Comment:
Only just noticing this is in the network.proto. Is this because we receive both a peer and advert message on solicit? If its not the case then this should surely be in the router.proto and fired over the control channel?

Reply:
Looking at the `processCtrlChan()` I'm thinking it makes more sense to move the `Solicit` message into `router.proto`, actually 🤔Will do that.

Reply:
Yea think it makes sense to move it to router.proto and use in control channel if its related to returning routes

Reply:
Working on this now.